lib/plugins/create/templates/hello-world/serverless.yml

```diff
@@ -21,4 +21,3 @@ functions:
       - http:
           path: hello-world
           method: get
-          cors: true
```

Comment:
Could you or @DavidWells elaborate why? All my tests worked just fine without CORS enabled.

I do think it is great to have a nice simple way to enable CORS and show that. But if we show it we should also explain why and have the example / test highlight that feature. IMHO a helloworld should be as small as possible. So I would leave CORS out.